src/script/repositories/entity/message/Message.ts

```diff
@@ -192,13 +192,24 @@ export class Message {
    * @returns Message contains text
    */
   hasAssetText(): boolean {
-    return this.isContent() ? this.assets().some(assetEntity => assetEntity.isText()) : false;
+    return this.isContent()
+      ? this.assets().some(assetEntity => assetEntity.isText() || assetEntity.isMultipart())
+      : false;
   }
 
   hasMultipartAsset(): boolean {
     return this.isContent() ? this.assets().some(assetEntity => assetEntity.type === AssetType.MULTIPART) : false;
   }
 
+  getMultipartAssets() {
+    const hasMultipartAsset = this.hasMultipartAsset();
+    if (!hasMultipartAsset || !this.isContent()) {
```

Comment:
No need of  `!this.isContent()` as it is already checked inside `hasMultipartAsset`
```suggestion
    if (!hasMultipartAsset) {
```